src/libraries/Microsoft.PowerFx.Core/Syntax/Visitors/DefaultVisitor.cs

```diff
@@ -101,6 +101,16 @@ public override TResult Visit(TableNode node, TContext context)
         }
 
         public override TResult Visit(AsNode node, TContext context)
+        {
+            return Default;
+        }
+
+        public override TResult Visit(StrInterpNode node, TContext context)
+        {
+            return Default;
+        }
+
+        public override TResult Visit(TypeLiteralNode node, TContext context)
```

Comment:
>TResult Visit(TypeLiteralNode node, [](http://example.com/codeflow?start=24&length=35)

Don't you need to update TexlFunctionalVisitor definition for it?

Reply:
resolved: clarified offline